src/app/events/[slug]/utils/scrollRefIntoView.ts

```diff
@@ -0,0 +1,3 @@
+export function scrollRefIntoView(ref: React.RefObject<HTMLElement>) {
```

Comment:
It wasn't long ago that we implemented `scrollToTabGroup` and yet both `scrollToTabGroup` and `scrollRefIntoView` made me wonder what this was for. What about `scrollToTabContentStart`? Or something like that?

Reply:
Sure, that works :)